### ISSUE TITLE
Auto assign roles is illogical

### ISSUE THREAD

**Issue by maixuanhan** [FauDroids/Werewolf#3]
If the **Number of Players** is even, by default the number of werewolves is always equal to the number of villagers. It's definitely the criteria of werewolf team winning.

**Comment by Maddoc42** [comment]
Fair point. Right now auto assign ratio is 1 : 1. Are there any rules of thumbs what makes a good distribution? How about 1 : 2 with 1 part werewolves and 2 parts villagers?

**Comment by maixuanhan** [comment]
you decide
